src/app/nft-showcase/nft-showcase.component.ts

```diff
@@ -3,6 +3,7 @@ import { BackendApiService, NFTCollectionResponse } from "../backend-api.service
 import { GlobalVarsService } from "../global-vars.service";
 import { InfiniteScroller } from "../infinite-scroller";
 import { IAdapter, IDatasource } from "ngx-ui-scroll";
+import * as _ from "lodash";
```

Comment:
We should probably just import the uniqBy function here to optimize the size of the bundle